docker-compose.yml

```diff
@@ -115,7 +115,7 @@ extractorapp:
     - smtp
   volumes:
     - /etc/georchestra:/etc/georchestra
-    - extractorapp_extracts:/var/local/extracts
+    - extractorapp_extracts:/mnt/extractorapp_extracts
```

Comment:
to be synced with : https://github.com/georchestra/georchestra/blob/master/extractorapp/src/docker/Dockerfile#L15 ?